CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
         - WasteWorks PWA can now have a separate name from the FixMyStreet PWA.
```

Comment:
Question: Should I be adding report_age() changes for Northumberland in this PR, or do that separately?

Reply:
Yep, happy to have it here :)

Reply:
Have realised I am not clear on what timeframes, for what states specifically, Northumberland want in https://mysocietysupport.freshdesk.com/a/tickets/3097. 
Seems to change from 'Can we remove closed reports after 5 working days of us closing?' to 'We could do with a change when the state has been changed to fixed closed etc[...]If pins could be removed if no further actions are required it would be good. Say 10 working days upon completion.'
I.e. what started in my interpretation as { closed => '5 days' } becomes { closed => '10 days', fixed => '10 days' }.

Also it may not be clear to Northumberland that we're only implementing this for upper-level states of open/closed/fixed; Kris mentions things like 'Maybe show open, investigating and action scheduled.' though tbh those are all captured under the wider umbrella of 'open' and if that also includes 'for triage' etc. I doubt that's going to be much of an issue. I can leave a comment for Kris requesting clarification.

Reply:
Given that ticket, I wouldn't bother requesting clarification, just introduces delays and possibility of further confusion, I would go with closed/fixed of "14 days" ~ equal 10 working days :)

Reply:
OK 👍 They can say whether they're happy with it after the fact.